.github/workflows/slack-pr-reminder.yml

```diff
@@ -1,37 +1,19 @@
-name: Notify Pending PR Reviewers
+name: Daily PR Review Reminder
 
 on:
-  workflow_dispatch:
   schedule:
-    - cron: '0 16 * * 1-5'  # 11:00 AM CT M-F
+    - cron: "0 16 * * 1-5" # 11:00 AM CT M-F
+  workflow_dispatch:
 
 jobs:
   notify-reviewers:
     runs-on: ubuntu-latest
     if: github.repository == 'DefectDojo/django-DefectDojo' # Notify only in core repo, not in forks - it would just fail in fork
-
     steps:
-      - name: Checkout repository
-        uses: actions/checkout@v4
+      - name: Notify reviewers in Slack
+        uses: DefectDojo-Inc/notify-pr-reviewers-action@master
```

Comment:
It would be nicer to have it pinned to hash and git tag

Reply:
I actually left that off intentionally since DDInc maintains that repo and it is access controlled